views/system.ejs

```diff
@@ -73,28 +79,48 @@
 
                                         </div>
                                         <div class="col-12 col-sm-12 col-md-6 col-lg-3 col-xl-3 text-center">
-                                            <p class=""><i class="fas fa-hourglass-half"></i> OctoFarm Uptime: <br><span class="text-light" id="processUpdate" style="font-size:0.8rem;">
+                                            <div class="row">
+                                                <div class="col-12 col-sm-12 col-md-6 col-lg-6 col-xl-6 text-center">
+                                                    <p class=""><i class="fas fa-hourglass-half"></i> OctoFarm Uptime: <br><span class="text-light" id="processUpdate" style="font-size:0.8rem;">
                                                            <%= helpers.generateTime(systemInformation.processUptime) %>
-                                        </span>
+                                                    </span>
 
-                                            </p>
-                                            <p><i class="fas fa-code-branch"></i> Version:                                     <a title="OctoFarm Version, will take you to github changelog" class="text-light"
-                                                                                                                                  href="https://github.com/NotExpectedYet/OctoFarm/releases"
-                                                                                                                                  target="_blank"><br><%= version %></a></p>
-                                            <!--                                    <p><i class="fas fa-code-branch"></i> Latest Version:                                     <a title="OctoFarm Version, will take you to github changelog" class="text-light"-->
-                                            <!--                                                                                                                          href="https://github.com/NotExpectedYet/OctoFarm/releases"-->
-                                            <!--                                                                                                                          target="_blank"></a></p>-->
-                                            <p class=""><i class="far fa-clock"></i> Timezone: <br><span class="text-light" id="timeZone" style="font-size:0.8rem;">
+                                                    </p>
+                                                    <p><i class="fas fa-code-branch"></i> Version: <a title="OctoFarm Version, will take you to github releases page" class="text-light"
```

Comment:
Not the latest release? We get an URL with the release response :)

Reply:
Aye will check your code a bit more and add that good idea.